dev-java/jetty-util/jetty-util-9.3.5.20151012.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="5"
+
+JAVA_PKG_USE="doc source"
+
+inherit java-pkg-2 java-pkg-simple
+
+DESCRIPTION="Jetty's Utils"
+
+MY_PN="jetty"
+MY_PV="${PV/2015/v2015}"
+MY_P="${MY_PN}-${MY_PV}"
+
+SLOT="$(get_version_component_range 1-2)"
```

Comment:
I do not believe versionator can be removed from java-utils-2, and if it was and broke ebuilds. That would be the result of an eclass change, and they should not be modified to break things. This entire thing is based off the chance that versionator might be dropped from java-utils-2. Which is not very likely ever. It has just as much of a chance of being dropped, as java-pkg-2 and others no longer inheriting java-utils-2. You cannot say otherwise. Chance is the same for both. With that being the case, either all java ebuilds need to inherit java-utils-2, or we can rely on versionator inherit always being part of java-utils. Last I checked less code in ebuilds is better than more, so if we have one less inherit because the eclass is, that is a benefit. One that maybe should not ever be changed. In addition to the eclasses own direct usage.